src/domain/club/club.go

```diff
@@ -0,0 +1,24 @@
+package club
+
+import "context"
+
+type ClubDBIF interface {
+	Create(ctx context.Context, club *Club) (res *Club, err error)
+	Update(ctx context.Context, club *Club) (res *Club, err error)
+	FindByID(ctx context.Context, clubID string) (res *Club, err error)
+}
+
+type Club struct {
+	ID             string
+	Name           string
+	ApprovalStatus string
+	ClubMembers    []ClubMember
```

Comment:
```suggestion
	id             string
	name           string
	approvalStatus string
	clubMembers    []ClubMember
```

全てprivateですね。

publicにしてしまうと、setterをつけてしまうことになるため、ドメインルールでカプセル化した構造体も、自由に使う側に変更されてしまいます。

Reply:
またそれに合わせてgetterも必要になります。

例えばIDだと
```
func (c *Cluc) ID() string {
  return c.id
}
```

このようなgetterになります。

Reply:
修正しました！ https://github.com/maloweb3/golang-school-app-sample/pull/1/commits/a96695c70ce5c7fd7c05fd65a88b2e070074ee11